### ISSUE TITLE
Is there a problem in focal_loss?

### ISSUE THREAD

**Issue by StrongerzZ** [ponta256/fssd-resnext-voc-coco#1]
Thank you for the code.
I was a little puzzled when checking focal loss:
         x = torch.ones (target.shape) * 0.25
         y = torch.ones (target.shape) * 0.75
         alpha = torch.where (target == 0, x, y)
Should it be:
         x = torch.ones (target.shape) * 0.75
         y = torch.ones (target.shape) * 0.25
         alpha = torch.where (target == 0, x, y)
Maybe I was wrong. Can you help me? @ponta256 
thanks a lot!

**Comment by ponta256** [comment]
@StrongerzZ 
Sorry for my late reply. I've just noticed your message. Does the code make sense for you now? 
I don't recall the details off the top of my head but you still think it could be my mistake, I am happy to look into it. Thanks!

**Comment by StrongerzZ** [comment]
I checked the relevant papers and found that you were right.
I am still waiting for the final training result.
Thanks again :)

**Comment by ponta256** [comment]
@StrongerzZ 
Great to hear that. Hope it goes well. : )
